src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AttributeNamedParameterCompletionProviderTests.cs

```diff
@@ -107,7 +103,8 @@ public class TestAttribute : Attribute
     public string Text { get; set; }
 }";
 
-            await VerifyItemExistsAsync(markup, "Text", displayTextSuffix: " =");
+            using var workspaceFixture = CreateWorkspaceFixture();
```

Comment:
> Can we move this creation/destruction to the class constructor so it's not being repeated for every single test?

We cannot move destruction past the logical end of this method. It could be moved in theory into the call(s) made by the test, but it cannot be lifted out to the containing class.

> Could we then move into the base classes so that way we also don't have to pass it for each VerifyItemExistsAsync?

This is possibly viable, but then you go from having one workspace used for several assertions to one workspace for _each_ assertion. This code is a closer match to the original behavior of the test code. I'm not sure it makes a difference in the end.

Reply:
> Can we move this creation/destruction to the class constructor so it's not being repeated for every single test

➡️ I rewrote this using `ReferenceCountedDisposable<TWorkspaceFixture>` which avoids the need to deal with the fixture except in tests that use it directly.